GPCS4/Util/UtilSync.h

```diff
@@ -11,38 +11,55 @@
  * protect data structures for a short duration
  * in case the structure is not likely contested.
  */
-class Spinlock 
-{
 
-public:
+#ifdef _M_X64
+#define __UTIL_SYNC_X86__ 1
+#elif _M_IX86
+#define __UTIL_SYNC_X86__ 1
+#elif __x86_64
+#define __UTIL_SYNC_X86__ 1
+#elif __i386
+#define __UTIL_SYNC_X86__ 1
+#endif
```

Comment:
we might want custom behavior on other arches. remove this so we get a compile time error if it happens

Reply:
done